lighthouse-core/test/fixtures/wikipedia-redirect.critical-request-chains.json

```diff
@@ -1,11 +1,11 @@
 {
-  "33097.1:redirected.0": {
```

Comment:
open question if we want to match exactly the old API instead?

Reply:
i dun care. this wfm.